Code/training_common_utils.py

```diff
@@ -141,10 +147,11 @@ def training_loop(
 
         model.train()
         # assuming loader will return image, label tuple
-        print("The type of training_loader is:",type(training_loader))
+        print("The type of training_loader is:", type(training_loader))
         for i, (images, labels) in enumerate(training_loader):
             print("\tMiniBatch {}:".format(i))
             minibatch_start = time.time()
+            #print("labels:", labels)
```

Comment:
added this for debugging (was getting nan loss values again. 
then commented it out for more reasonable logging.